File: src/main/java/com/rkroom/blog/entity/Site.java
package com.rkroom.blog.entity;

import javax.persistence.*;

@Entity
public class Site {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    @Column(columnDefinition="varchar(191) not null",unique = true,nullable = false)
    private String attribute;
    @Column(columnDefinition="text")
    private String value;
    private boolean authorization;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getAttribute() {
        return attribute;
    }

    public void setAttribute(String attribute) {
        this.attribute = attribute;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    public boolean isAuthorization() {
        return authorization;
    }

    public void setAuthorization(boolean authorization) {
        this.authorization = authorization;
    }
}

File: src/main/java/com/rkroom/blog/entity/Site.kt
package com.rkroom.blog.entity

import javax.persistence.*

@Entity
data class Site(    @Id
                    @GeneratedValue(strategy = GenerationType.IDENTITY)
                    var id: Int? = null,
                    @Column(columnDefinition = "varchar(191) not null", unique = true, nullable = false)
                    var attribute: String? = null,
                    @Column(columnDefinition = "text")
                    var value: String? = null,
                    var isAuthorization: Boolean = false)